### PR TITLE
added lobby id component to the scribble page

### PR DESCRIPTION
**Pull request by alantylam**

### CODE REVIEW

**Comment by alantylam** [comment]
Arcade - Mozilla Firefox 2020-07-25 3_21_56 PM (2)

File: frontend/src/modules/scribble/components/LobbyID.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="lobby-id-section">
```
Comment (gordon-cheung) [inline frontend/src/modules/scribble/components/LobbyID.vue:2]: Can you check the vue to be LobbyId.vue, and then this div should be the name of the component (lobby-id)

File: frontend/src/modules/scribble/components/LobbyID.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                        readonly
+                /></b-col>
+            </b-row>
+        </b-container>
+    </div>
+</template>
+
+<script>
+export default {
+    name: "LobbyID",
```
Comment (gordon-cheung) [inline frontend/src/modules/scribble/components/LobbyID.vue:20]: Also here

File: frontend/src/modules/scribble/components/LobbyID.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="lobby-id-section">
+        <b-container fluid>
+            <b-row align-v="center" class="justify-content-md-center">
+                <b-col md="auto">Lobby ID: </b-col>
+                <b-col md="auto"
+                    ><input
+                        id="lobby-id"
```
Comment (gordon-cheung) [inline frontend/src/modules/scribble/components/LobbyID.vue:8]: Lets ur class instead of id here, following BEM it should be lobby-id__input

**Pull request merged**